Client/Frontend/Browser/Browser.swift

```diff
@@ -6,7 +6,6 @@ import Foundation
 import WebKit
 
 protocol BrowserHelper {
-    init?(browser: Browser)
```

Comment:
ReaderMode is an implementation that uses it. But it does not matter. I think we can leave it out.